static/js/actions/ui.js

```diff
@@ -128,3 +128,6 @@ export const setNavDrawerOpen = createAction(SET_NAV_DRAWER_OPEN);
 
 export const SET_LEARNER_CHIP_VISIBILITY = 'SET_LEARNER_CHIP_VISIBILITY';
 export const setLearnerChipVisibility = createAction(SET_LEARNER_CHIP_VISIBILITY);
+
+export const SET_TO_PEARSON_SITE_DIALOG_VISIBILITY = 'SET_TO_PEARSON_SITE_DIALOG_VISIBILITY';
```

Comment:
In fact, instead of creating a new action, I would suggest you use the `SHOW_DIALOG` / `HIDE_DIALOG` pattern (we're using it currently only for the email dialogs I think, but it's a bit cleaner overall).